tests/p2p/test_p2p.py

```diff
@@ -155,8 +191,8 @@ def test_many_gossip_messages(setup_clients, add_curl):
 
     # *note*: this already waits for bootstrap so we can send the msg right away.
     # send message to client via rpc
-    TEST_MESSAGES = 10
-    for i in range(TEST_MESSAGES):
+    test_messages = 10
```

Comment:
constant should be in CAPITAL

Reply:
it's a variable not a constant

Reply:
fixed in 207fd37bd289dcc691ce4b4207a102bc0f009af7